kernel/vmvalues.ml

```diff
@@ -683,3 +695,15 @@ let parray_get_default = Obj.magic Parray.default
 let parray_set = Obj.magic Parray.set
 let parray_copy = Obj.magic Parray.copy
 let parray_length = Obj.magic Parray.length
+
+let pchar_code = Obj.magic Pstring.Char.code
+let pchar_chr = Obj.magic Pstring.Char.chr
+
+let pstring_make = Obj.magic Pstring.make
+let pstring_length = Obj.magic Pstring.length
+let pstring_get = Obj.magic Pstring.get
+let pstring_sub = Obj.magic Pstring.sub
+let pstring_cat = Obj.magic Pstring.cat
+let pstring_equal = Obj.magic Pstring.equal
+let pstring_lt = Obj.magic Pstring.lt
+let pstring_le = Obj.magic Pstring.le
```

Comment:
If I remember correctly, Coq's and OCaml's boolean values are encoded in opposite directions. So, extra work has to be performed here, something like
```ocaml
let pstring_equal = Obj.magic (fun x y -> if Pstring.equal x y then 0 else 1)
```

Reply:
We should make sure there's some code in the test suite that ensures we get this right

Reply:
Yeah, I'll add tests for all the primitives, but I first need to figure out how to extend the notation mechanism to get literals. Doing anything without that is way too painful.

Reply:
I added tests.